tests/testthat/test-collate.R

```diff
@@ -47,4 +47,9 @@ test_that("DESCRIPTION file is re-written only if collate changes", {
 
  })
 
-
+test_that("Collate with one file name per line (#790)", {
+  expect_equal(
+    roxygen2:::package_files('testCollateParse'),
```

Comment:
You can omit the  `roxygen2:::` here I think.

Reply:
Done.